compiler/src/iree/compiler/Codegen/SPIRV/SPIRVVectorizeLoadStore.cpp

```diff
@@ -37,13 +37,17 @@ constexpr int kMaxVectorNumElements = 4;
 
 namespace mlir {
 namespace iree_compiler {
+namespace {
+
+//===----------------------------------------------------------------------===//
+// Utility Functions
+//===----------------------------------------------------------------------===//
```

Comment:
nit: @benvanik and I had a discussion about this a few months ago and we converged on keeping functions static instead of moving them to anonymous namespaces. This is because it's easier to tell by just looking at the definition and more difficult to mess up when moving namespaces around. I believe this is also the state of things in mlir.

Reply:
I don't have a strong preference here; I was trying to silence the warnings from clang-tidy in the category of "misc-use-anonymous-namespace". Maybe we should copy the LLVM's `.clang-tidy` to IREE root so we inherit some configurations there.

Reply:
Reverted back to the original form.